README.md

```diff
@@ -189,23 +188,15 @@ Some things to note:
 - `max_fee` defaults to `0`. Add `--max_fee <max_fee>` to set the maximum fee for the transaction
 - `network` defaults to the `localhost`. Add `--network <network>` to change the network for the transaction
 
-### `call` and `invoke`
+### `call`
 
-Using `call` and `invoke`, we can perform read and write operations against our local node (or public one using the `--network mainnet` parameter). The syntax is:
+Using `call`, we can perform read operations against our local node or the specified public network. The syntax is:
```

Comment:
we could add a mention to `simulate` and `estimate_fee` here or in the other PR

Reply:
I added a mention for those in the other PR under the `send` section, are you suggesting adding it under `call` too?

Reply:
yes that was the suggestion but i don't think it's a good idea anymore